kubetest/kubeadmdind/kubeadm_dind.go

```diff
@@ -402,7 +402,7 @@ func (d *Deployer) clusterSize() (int, error) {
 	return 0, nil
 }
 
-// Create a local log artifacts directory
```

Comment:
This is not a valid GoDoc comment.

Reply:
/lgtm cancel

Reply:
eh, why not?

Reply:
Sorry, I misspoke, this is valid, it is just bad style/convention. From effective go:
> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

> If every doc comment begins with the name of the item it describes, the output of godoc can usefully be run through grep. Imagine you couldn't remember the name "Compile" but were looking for the parsing function for regular expressions, so you ran the command, `$ godoc regexp | grep -i parse`
If all the doc comments in the package began, "This function...", grep wouldn't help you remember the name. But because the package starts each doc comment with the name, you'd see something like this, which recalls the word you're looking for.

So the original comment was more useful.